src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,140 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exeption.UserInvalidExeption;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final String VALID_LOGIN = "John_Doe";
+    public static final String VALID_USERPASSWORD = "strongPassword";
+    public static final String SHORT_LOGIN = "Joe";
+    public static final String SHORT_PASSWORD = "abcde";
+    public static final int VALID_USERAGE = 25;
+    private RegistrationServiceImpl registrationService;
+    private StorageDaoImpl storageDao;
+    private User validUser;
+
+    @BeforeEach
+    void setUp() {
+        validUser = new User();
+        validUser.setAge(VALID_USERAGE);
+        validUser.setLogin(VALID_LOGIN);
+        validUser.setPassword(VALID_USERPASSWORD);
+        registrationService = new RegistrationServiceImpl();
```

Comment:
we do not need a new service before each test, we can intialize it once in the `BeforeAll` or just straight up in the field, both variants are fine

Reply:
Ok, tnx.